README.md

```diff
@@ -142,3 +142,37 @@ directory. This is the only directory you need to preserve.
 
 > **Note:** the name of the final Git repository can be changed using the `-repo`
 > flag.
+
+### Pushing new versions to kubewarden/k8s-objects
+
+Set `KUBERNETES_VERSION_MIN`, `KUBERNETES_VERSION_MAX` in `mass-generate.sh`,
+`KUBEMINOR` in `mass-push.sh` as needed.
+
+Delete and reclone kubewarden/k8s-objects. In my case:
```

Comment:
You don't have to do that, you just need the repo checked out locally.

Reply:
Amended.